sdk/python/tests/compiler/testdata/volume_snapshot_op.log

```diff
@@ -0,0 +1,5 @@
+[create-volume : main] time="2020-06-09T02:51:39Z" level=error msg="Initialize script failed: exit status 2:"
+[create-volume : main] time="2020-06-09T02:51:39Z" level=info msg="kubectl create -f /tmp/manifest.yaml -o json"
+[create-volume : main] time="2020-06-09T02:51:39Z" level=error msg="Run kubectl command failed with: exit status 1 and Error from server (Forbidden): error when creating \"/tmp/manifest.yaml\": persistentvolumeclaims is forbidden: User \"system:serviceaccount:kubeflow:default\" cannot create resource \"persistentvolumeclaims\" in API group \"\" in the namespace \"kubeflow\""
+[create-volume : main] time="2020-06-09T02:51:39Z" level=error msg="Execute resource failed: exit status 1:"
```

Comment:
Snapshot op right now only works on minikube because it's a k8s alpha feature that requires a feature gate from k8s master.